--- a/drivers/media/usb/dvb-usb/dw2102.c	2017-04-17 21:00:11.000000000 +0100
+++ b/drivers/media/usb/dvb-usb/dw2102.c	2017-09-27 23:13:43.195766409 +0100
@@ -1714,7 +1714,8 @@
 	TECHNOTREND_S2_4600,
 	TEVII_S482_1,
 	TEVII_S482_2,
-	TEVII_S662
+	TEVII_S662,
+	TERRATEC_CINERGY_S2_BOX,
 };
 
 static struct usb_device_id dw2102_table[] = {
@@ -1746,6 +1747,7 @@
 	[TEVII_S482_1] = {USB_DEVICE(0x9022, 0xd483)},
 	[TEVII_S482_2] = {USB_DEVICE(0x9022, 0xd484)},
 	[TEVII_S662] = {USB_DEVICE(0x9022, USB_PID_TEVII_S662)},
+	[TERRATEC_CINERGY_S2_BOX] = {USB_DEVICE(USB_VID_TERRATEC, 0x0105)},
 	{ }
 };
 
@@ -1903,7 +1905,7 @@
 		}},
 		}
 	},
-	.num_device_descs = 3,
+	.num_device_descs = 4,
 	.devices = {
 		{"DVBWorld DVB-S 2102 USB2.0",
 			{&dw2102_table[CYPRESS_DW2102], NULL},
@@ -1917,6 +1919,10 @@
 			{&dw2102_table[TERRATEC_CINERGY_S], NULL},
 			{NULL},
 		},
+		{ "Terratec Cinergy S2 USB BOX",
+			{ &dw2102_table[TERRATEC_CINERGY_S2_BOX], NULL },
+			{ NULL },
+		},
 	}
 };
 
