diff --git a/drivers/media/v4l2-core/v4l2-compat-ioctl32.c b/drivers/media/v4l2-core/v4l2-compat-ioctl32.c
--- a/drivers/media/v4l2-core/v4l2-compat-ioctl32.c
+++ b/drivers/media/v4l2-core/v4l2-compat-ioctl32.c
@@ -20,6 +20,10 @@
 
 #ifdef CONFIG_COMPAT
 
+#define put_compat_timespec(ts, cts)                            \
+                (put_user((ts)->tv_sec, &((cts)->tv_sec))) ||    \
+                 put_user((ts)->tv_nsec, &((cts)->tv_nsec))
+
 static long native_ioctl(struct file *file, unsigned int cmd, unsigned long arg)
 {
 	long ret = -ENOIOCTLCMD;
